patches/chrome-browser-notifications-notification_platform_bridge_linux.cc.patch

```diff
@@ -1,12 +0,0 @@
-diff --git a/chrome/browser/notifications/notification_platform_bridge_linux.cc b/chrome/browser/notifications/notification_platform_bridge_linux.cc
-index d895f59a259935d5bd0193e8ab0fbd0bd4725d48..81b3610920411e5909fe9d269079ed27c7e8f3b4 100644
---- a/chrome/browser/notifications/notification_platform_bridge_linux.cc
-+++ b/chrome/browser/notifications/notification_platform_bridge_linux.cc
-@@ -658,6 +658,7 @@ class NotificationPlatformBridgeLinuxImpl
-       actions.push_back("Activate");
-       // Always add a settings button for web notifications.
-       if (notification_type != NotificationHandler::Type::EXTENSION &&
-+          notification_type != NotificationHandler::Type::BRAVE_ADS &&
```

Comment:
cc @tmancey - I don't think this is supposed to be removed here

Reply:
The current Chromium code is
<pre>
actions.push_back("Activate");
      // Always add a settings button for web notifications.
      if (notification->should_show_settings_button()) {
        actions.push_back(kSettingsButtonId);
</pre>

and 
<pre>
bool should_show_settings_button() const {
    return optional_fields_.settings_button_handler !=
           SettingsButtonHandler::NONE;
  }
</pre>

I see `settings_button_handler` initialized to `NONE` and I don't see it set anywhere else.

Reply:
then I think we can also remove the patch in notification_platform_bridge_mac.mm

Reply:
Removed in 9ea29de4faa6b5ca974a291a60bfcdcef25aacab